### DISCUSSION TITLE
Keycloak documentation is missing

### DISCUSSION THREAD

**Discussion by Rezzhaf** [keycloak/keycloak#15267] [Q&A]
Dear All dev

so, yesterday i was trying to install operator for keycloak with this link of documentation :

https://www.keycloak.org/docs/latest/server_installation/index.html#_install_by_command
https://www.keycloak.org/getting-started/getting-started-operator-kubernetes

but today i want to continue my research but the link is off/broken

is there something goin on?

**Comment by Rezzhaf** [comment]
i really apreciate it if someone can help

**Comment by stianst** [comment]
https://www.keycloak.org/guides#operator

**Reply by Rezzhaf** [reply]
actually yesterday im in the middle of installation

https://www.keycloak.org/docs/latest/server_installation/index.html#_install_by_command

is there any other link? i use keycloak on kubernetes

**Reply by Rezzhaf** [reply]
because i use vanilla kubernetes its seems doenst work

**Reply by stianst** [reply]
That link is for the deprecated WildFly distribution, which was removed in Keycloak 20. Please check the updates guides I've linked to for the new operator.

If you still want to look at the old (and now no longer supported) operator go to the website and find the docs for 19.0.

**Reply by Rezzhaf** [reply]
old documentation still linked to version 20

**Reply by Rezzhaf** [reply]
https://www.keycloak.org/docs/latest/server_installation/

latest is not avaible
